projects/spectator/jest/test/highlight.directive.spec.ts

```diff
@@ -24,3 +24,25 @@ describe('HighlightDirective', () => {
     });
   });
 });
+
+describe('HighlightDirective (createHostDirectiveFactory)', () => {
+  let host: SpectatorForDirective<HighlightDirective>;
+
+  const createHost = createHostDirectiveFactory(HighlightDirective);
+
+  // calculated styles not supported in JSDOM
+  xit('should change the background color', () => {
```

Comment:
remove xit

Reply:
Oh, I removed it but the tests fail. I had to exclude these tests earlier because the `toHaveStyle()` matcher is not supported in Jest. It relies on `getComputedStyle().getPropertyValue()`.

Reply:
There is a way to check if the user uses Jest and warn them that it will not work?

Reply:
We currently don't have a way to detect it. Let's see if another solution is possible, e.g. not relying on computed styles. Let's fix it in a separate PR.